packages/core/src/routes/types.ts

```diff
@@ -3,6 +3,10 @@ import Router from 'koa-router';
 import { WithAuthContext } from '@/middleware/koa-auth';
 import { WithI18nContext } from '@/middleware/koa-i18next';
 import { WithUserInfoContext } from '@/middleware/koa-user-info';
+import { WithUserLogContext } from '@/middleware/koa-user-log';
 
-export type AnonymousRouter = Router<unknown, WithI18nContext>;
-export type AuthedRouter = Router<unknown, WithUserInfoContext<WithAuthContext<WithI18nContext>>>;
+export type AnonymousRouter = Router<unknown, WithUserLogContext<WithI18nContext>>;
+export type AuthedRouter = Router<
+  unknown,
+  WithUserInfoContext<WithAuthContext<WithUserLogContext<WithI18nContext>>>
```

Comment:
`AuthedRouter` doesn't mean we have `UserLogContext`, this'll float in the future.

Reply:
same as above, (LOG-454)